src/core/1/plugins/guides/controllers/index.md

```diff
@@ -48,6 +48,72 @@ In order to create a new controller, the plugin must expose the following proper
 
 ---
 
+## Return a response
+
+By default, Kuzzle wraps the value returned by the function associated with the controller's action into the `result` property of the [Kuzzle Response](/core/1/api/essentials/kuzzle-response/).  
+
+Consider the following action:
+```js
+async init (config, context) {
+  this.controllers = {
+    myController: {
+      saySomething: 'saySomething'
+    }
+  }
+};
+
+async saySomething (request) {
+  return {
+    foo: 'bar'
+  };
+}
+```
+
+When this action is successfully called, the following answer will be returned by Kuzzle:
+
+```json
+{
+  "requestId": "32dfbe90-34e1-43c0-a857-25b715b28a1b",
+  "status": 200,
+  "error": null,
+  "controller": "myPlugin/myController",
+  "action": "saySomething",
+  "volatile": null,
+  "result":
+  {
+    "foo": "bar"
+  }
+}
+```
+
+However, it is possible to tell Kuzzle not to wrap the returned value in a Kuzzle Response and send it as it is.  
+To do this, use the [Request.setResult](/core/1/plugins/plugin-context/constructors/request/#setresult) method  specifying the `raw: true` option.
```

Comment:
(optional) A more straightforward way of doing that without the troublesome "null" result:

```js
request.response.raw = true;
request.response.headers['Content-Type'] = 'application/json';
```